action/lib/deduplicate.js

```diff
@@ -20,9 +20,11 @@ export default async function ({ octokit, workflow_id, run_id }) {
   // filter and sort
   const cancellable = workflow_runs
     // filter to relevant runs
-    .filter(run => ['in_progress', 'queued'].includes(run.status))
+    .filter(run => ['in_progress', 'queued', 'completed'].includes(run.status))
```

Comment:
these are all the available types, so this check is no longer relevant, can delete this whole line

Reply:
done